content/docs/bdd/better-gherkin.md

```diff
@@ -35,3 +35,44 @@ In fact, a good question to ask yourself when writing a feature clause is: “Wi
 
 If the answer is “Yes”, then you should rework it avoiding implementation specific details.
 As a side benefit, in consequence your scenarios will be a lot shorter and much easier to follow and understand.
+
+# Consider a more declarative style
+
+One way to make tests easier to maintain and less brittle is to use a declarative style. Declarative style describes the behaviour of the application, rather than the implementation details. Declarative tests read better as "living documentation". A declarative style helps you focus on the value that the customer is getting, rather than the keystrokes they will use.
```

Comment:
```suggestion
One way to make scenarios easier to maintain and less brittle is to use a declarative style. Declarative style describes the behaviour of the application, rather than the implementation details. Declarative tests read better as "living documentation". A declarative style helps you focus on the value that the customer is getting, rather than the keystrokes they will use.
```

Reply:
Done.